src/applications/personalization/view-dependents/components/ViewDependentsFormStates/ViewDependentsChildForm.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+
+function ViewDependentsChildForm() {
+  const schema = {
+    type: 'object',
+    properties: {
+      firstName: {
+        type: 'string',
+      },
+    },
+  };
+
+  const uiSchema = {};
+
+  function show() {}
+
+  return (
+    <SchemaForm
+      name="View Dependents Child Form"
+      title=""
+      schema={schema}
+      uiSchema={uiSchema}
+      onSubmit={show}
+    />
+  );
```

Comment:
Using `SchemaForm` directly is really cool. The only thing to watch out for is that our Forms System extends it with some helper logic, so by default you won't have all of the additional features you may be use to. I would just store the form state in your Redux store and manually call the functions to get the features. Here's an example, `src/applications/coronavirus-vaccination/reducers/index.js`

Reply:
yeah we will be adding to this in subisquent tickets